src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -184,6 +184,8 @@ protected void handleUserException(Exception e) {
 
         if(printStackTraceOnUserExceptions()) {
             e.printStackTrace();
+        } else {
+            System.out.println("Use -DSTACK_TRACE_ON_USEREXCEPTION to print the stack trace.");
```

Comment:
I don't have a strong opinion either way, I can change the code if we have consensus we prefer stderr.